File: hoja7/persona.php
<?php
//error_reporting(E_ALL);
class Persona {
    public $nombre;
    public $edad;
    public $genero;
    public function presentar(){
        echo 'Hola soy '.$this->nombre. ', tengo '.$this->edad. ' y soy '.$this->genero."\n";
    }
}

$presentacion = new Persona;
$presentacion -> nombre = 'Lamyaa';
$presentacion -> edad = 20;
$presentacion -> genero = 'mujer.';
$presentacion -> presentar();
?>

File: Java4/Ejercicio_32/paquetePersona/Persona.java
package paquetePersona;

public class Persona {
	String nombre;
	int edad;
	
	public Persona() {
		this.nombre = "Desconocido";
		this.edad = 0;
		
		System.out.println("Me llamo " + nombre + ", tengo " + edad + " años.");
	}
	
	public Persona(String nombre) {
		this.nombre = "Desconocido";
		this.edad = 0;
		
		System.out.println("Me llamo " + nombre + ", tengo " + edad + " años.");

	}
	
	public Persona(String nombre, int edad) {
		this.nombre = "Desconocido";
		this.edad = 20;
		
		System.out.println("Me llamo " + nombre + ", tengo " + edad + " años.");

	}
}
